backend/coins/btc/handlers/handlers.go

```diff
@@ -171,11 +187,68 @@ func (handlers *Handlers) getAccountTransactions(_ *http.Request) (interface{},
 			}[txInfo.Type],
 			Status:    txInfo.Status,
 			Amount:    handlers.formatAmountAsJSON(txInfo.Amount, false),
-			Fee:       feeString,
 			Time:      formattedTime,
 			Addresses: addresses,
 			Note:      handlers.account.TxNote(txInfo.InternalID),
 		}
+		result = append(result, txInfoJSON)
+	}
+	return result, nil
+}
+
+func (handlers *Handlers) getTransaction(r *http.Request) (interface{}, error) {
```

Comment:
a lot in this function is now duplicated with the contents of the other function, `getAccountTransactions`.

I think it's worthwhile to try to refactor it so that the duplicated code is removed, i.e. a helper function that both functions call or similar.

Reply:
Agree :+1: